### PR TITLE
New package: kiten-20.04.2

### PR DESCRIPTION
**Pull request by basketdou**

### CODE REVIEW

File: srcpkgs/kiten/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'kiten'
+pkgname=kiten
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```
Comment (sgn) [inline srcpkgs/kiten/template:6]: `-DBUILD_TESTING=ON`

File: srcpkgs/kiten/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="extra-cmake-modules pkg-config gettext"
+makedepends="qt5-devel qt5-declarative qt5-declarative-devel
+khtml-devel ki18n-devel knotifications-devel kdoctools-devel kxmlgui-devel karchive-devel kcoreaddons-devel kcompletion-devel kconfig-devel kconfigwidgets-devel gettext-devel kcrash-devel"
+depends="khtml hicolor-icon-theme"
```
Comment (sgn) [inline srcpkgs/kiten/template:10]: `khtml` should be auto-detected.

File: srcpkgs/kiten/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'kiten'
+pkgname=kiten
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules pkg-config gettext"
+makedepends="qt5-devel qt5-declarative qt5-declarative-devel
+khtml-devel ki18n-devel knotifications-devel kdoctools-devel kxmlgui-devel karchive-devel kcoreaddons-devel kcompletion-devel kconfig-devel kconfigwidgets-devel gettext-devel kcrash-devel"
```
Comment (sgn) [inline srcpkgs/kiten/template:9]: This line is too long, please wrap at 72 characters.

File: srcpkgs/kiten/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+short_desc="Japanese Reference/Study Tool"
+maintainer="basketdou <basketdou@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://kde.org/applications/education/org.kde.kiten"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=c5d002ba2ee4a11913457808da67f9c7ee5dbcf1731fad11842a4aa338862c82
+
+pre_check() {
+export QT_QPA_PLATFORM=offscreen
+}
```
Comment (sgn) [inline srcpkgs/kiten/template:20]: indent with TAB, and move after the below `if`

File: srcpkgs/kiten/template [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" python3 qt5-host-tools qt5-qmake"
```
Comment (sgn) [inline srcpkgs/kiten/template:23]: I'll just merge this with above `hostmakedepends` regardless of `$CROSS_BUILD`

**Comment by sgn** [comment]
Also, the cross stuff is broken ;)

**Comment by ericonr** [comment]
Ping? I think this package could be merged, once suggestions are applied.

**Pull request closed without merging**